### PR TITLE
Updated the changes.

### PR DESCRIPTION
**Pull request by Mannxxx**
1. Updated README.md with Installation and Usage Instructions
2. Added a CONTRIBUTING.md file.
3. Added a Code_of_Conduct.md
4. Made changes to server.js and client.js files to make multiple users connect on the socket to share multiple files. (Maintainer Review required).

### CODE REVIEW

**Comment by Mannxxx** [comment]
@CS50X-RGB, Could you check it now?

File: README.md [hunk 1]
```diff
@@ -1,36 +1,95 @@
...
+
+3. Open a Pull Request (PR) to this repository.
+
+4. Our maintainers will review your PR and, if it meets the guidelines, merge it into the main project.
+
+## Our Valuable Contributors
+
+A big thank you to our contributors who help make this project a success! Add your name here by getting your PR merged.
+
+1. John Doe - [@johndoe](https://github.com/johndoe)
```
Comment (CS50X-RGB) [inline README.md:92]: Change the names here plz

Comment (CS50X-RGB) [reply README.md:92]: Add your name with your profile pic

**Review by CS50X-RGB — CHANGES_REQUESTED** [review]
Add the changes mentioned in the review plz add the rules I have provided you in the issues section

**Comment by CS50X-RGB** [comment]
@Mannxxx Rest all looks cool plz do the necessary changes

**Comment by Mannxxx** [comment]
@CS50X-RGB, Ohh wait, did you ask for adding a profile pic link too?

**Comment by CS50X-RGB** [comment]
Yeah yeah @Mannxxx

**Review by CS50X-RGB — APPROVED** [review]
@Mannxxx I have merged it good work :)

**Comment by CS50X-RGB** [comment]
Closes #4

**Pull request merged**